.github/workflows/build-image.yml

```diff
@@ -30,6 +30,8 @@ jobs:
           tags: superbench/main:rocm4.2-pytorch1.7.0
         - name: rocm4.0-pytorch1.7.0
           tags: superbench/main:rocm4.0-pytorch1.7.0
+        - name: rocm5.0-pytorch1.9.0
```

Comment:
could you keep the original ordering for name?

Reply:
sorry, should they be rocm5.0->rocm4.2->rocm4.0, updated